sktime_dl/deeplearning/base/estimators/_classifier.py

```diff
@@ -93,16 +93,25 @@ def check_and_clean_data(self, X, y=None, input_checks=True):
             else:
                 validate_X_y(X, y)
 
+        # want data in form: [instances = n][timepoints = m][dimensions = d]
         if isinstance(X, pd.DataFrame):
-            if X.shape[1] > 1 or not isinstance(X.iloc[0, 0], pd.Series):
-                raise TypeError(
-                    "Input should either be a 2d numpy array, or a pandas dataframe with a single column of Series objects (networks cannot yet handle multivariate problems")
+            if X.shape[1] > 1:
+                # we have multiple columns
+                # if cells contain series, this is a multidimensional problem
+                # else if cells contain single values, this is a univariate problem with values long columns
+                #      this situation can happen with e.g. forecasting-reduced-to-regression strategies
+                if isinstance(X.iloc[0, 0], pd.Series):
+                    # todo investigate incorporating the reshaping into the data extraction instead of this 2-stage process
+                    X = np.array([[X.iloc[r, c].values for c in range(len(X.columns))] for r in range(len(X))])
+                    X = X.reshape(X.shape[0], X.shape[2], X.shape[1])  # go from [n][d][m] to [n][m][d]
```

Comment:
I'd turn this routine into a utility function, it may also be covered already by the [tabularise](https://github.com/alan-turing-institute/sktime/blob/master/sktime/utils/data_container.py) helper function in sktime, can also be re-used in the classifier file below